CMakeLists.txt

```diff
@@ -113,9 +113,7 @@ if (ONEDPL_USE_PARALLEL_POLICIES)
             endif()
 
             # settings for the specific compilation type
-            if (NOT ONEDPL_USE_AOT_COMPILATION)
-                message(STATUS "Use the ahead of time compilation. No additional parameters needed")
```

Comment:
I think, we can leave this message and this if-branch and just fix "the ahead of time" to "the online". Because this option is a part of our CMake-interface and we have to print some info about it to our users.

Reply:
I thought that we should print only when users mark explicitly what they write. I think default behavior shouldn't be messaged. What are BKMs in other projects for that?